apps/emqx_resource/src/emqx_resource.erl

```diff
@@ -19,17 +19,13 @@
 -include("emqx_resource.hrl").
 -include("emqx_resource_utils.hrl").
 
+-elvis([{elvis_style, god_modules, #{limit => 33}}]).
```

Comment:
This module also requires splitting, but I think this should be done in a separate PR.